### DISCUSSION TITLE
I am having a build error for a github.io page.

### DISCUSSION THREAD

**Discussion by Hammed0616** [community/community#76938] [Actions] [answered]
### Select Topic Area

Question

### Body

I am trying to build a portfolio project and host it on github.io. I do not have any software development skills. So each time I try to deploy the page I get a 'Build error' message. Based on my little probing this is what I think is the cause of the error.

"Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss':
                    No such file or directory @ dir_chdir - /github/workspace/docs"

![image](https://github.com/community/community/assets/138397038/1aaf036b-7e89-4f3d-a189-b2cbf351e6cc)

Below is the link to the page.

https://github.com/Hammed0616/HammedPort.github.io

Thanks.

**Comment by subidit** [comment]
Rename your repository to `Hammed0616.github.io`.

**Reply by Hammed0616** [reply]
Thanks subidit,

I just did that but I am getting the same error message.

**Reply by subidit** [reply]
ok. then lets try to debug it step by step. 

go to https://github.com/Hammed0616/Hammed0616.github.io/settings/pages

do you have the following settings?
Build and deployment >
Source > Deploy from a branch
Branch > `main` and `/(root)` ?

If you had selected `docs` instead of `/(root)` it would give an error.

**Reply by Hammed0616** [reply]
Yes I selected docs and that was the option selected from the demonstration video I watched. Are you saying I should have selected /(root)?

**Reply by subidit** [reply]
yes. select `/(root)` .

in this case you should select root directory. the docs directory is used when you want to publish your project site from the docs directory.

**Reply by Hammed0616** [reply, marked as answer]
Wow! Thank you it worked

**Comment by kdinthehouse** [comment]
you need to make sure that the file 'assets/css/style.scss' is present. as i can see i dont see any file in your repo. it is important for Jekyll-based website

**Reply by Hammed0616** [reply]
Okay. But I download the folder from "html5 UP" so I don't which file is in there or not. 

How do you suggest I the "style.scss" in there

**Reply by kenturkey048** [reply]
below is the link of th repo..
please go through it.. I have my html file in a folder called public in the directory
https://github.com/kenturkey048/Space-Xproject

**Comment by sysserv** [comment]
**1 - Make sure that the file 'assets/css/style.scss' exists.
2 - Make sure that the file 'assets/css/style.scss' is in the same directory as the Jekyll configuration file.
3 - Check the Jekyll configuration file to make sure that it is correctly configured.
4 - Make sure that the source directory is set to the directory containing the Jekyll configuration file and the assets directory.**
